Faq

anxolin opened the pull request:
Closes #419 

## Test 
1. Go to FAQ page. You have a link in the secondary menu in the header
2. Read!

Content comes from https://docs.google.com/document/d/1_ze_Sv-9RwqHAIW_2sxPMbryQC0NpJRQ65z_rzrB7KY/edit#

W3stside commented:
@anxolin history is wrong.

alfetopito commented on src/custom/pages/Faq/Faq.md:
Shouldn't we have link on `Smart Contract Architecture`?

anxolin replied:
Makes sense. Pointing to the smart contracts for now. Probably good to use the dev portal later on.

alfetopito commented on src/custom/pages/Faq/Faq.md:
Either `are being added` or `will be added soon`

anxolin replied:
@annamsgeorge correcting this small detail. It's reviewed text by Kia, I hope is fine.

annamsgeorge replied:
thank you for noticing and correcting!

alfetopito commented on src/custom/pages/Faq/Faq.md:
```suggestion
This means, on CowSwap, when two traders each hold an asset the other wants, a trade can be settled directly between them without an external market maker or liquidity provider. This leads to better prices for the individual traders (because traditionally market makers add a fee — referred to as spread — for their surface). 
```

alfetopito commented on src/custom/pages/Faq/Faq.md:
Would be good to write when `now` is for a reference.

anxolin replied:
Agree, although the link already shows you the latest value

I would happily change it `now (April '20)`, and equally happy I can leave it as it is

cc/ @annamsgeorge

annamsgeorge replied:
I am glad you pointed this sentence out because it's incorrect. It's since January 2020 (!) not, 2021. 

I would suggest to rephrase to:

"Since January 2020 until now (April'21), the total amount ..."

alfetopito reviewed (approved):
Comments about the FAQ contents.
Should have done it before in the doc, I know. But we are here already, so...

anxolin commented:
> @anxolin history is wrong.

It was pointing to the wrong branch. now is fixed

annamsgeorge commented:
<img width="348" alt="Screenshot 2021-04-20 at 19 32 34" src="https://user-images.githubusercontent.com/33424780/115476123-7423a880-a241-11eb-9d62-273f1992108b.png">

I would add a bit more space before "Protocol" because it's a section header

annamsgeorge commented:
<img width="833" alt="Screenshot 2021-04-20 at 19 34 58" src="https://user-images.githubusercontent.com/33424780/115476242-c9f85080-a241-11eb-891c-beb65dd0d7fe.png">


This looks odd, as there is a 1) but no 2)...
Also the text is a bit weird, could we please update it to: (no need to put it in a box then) 

CowSwap can connect to all on-chain liquidity:
When CowSwap does not have enough CoWs (Coincidence of Wants) among the orders available for a batch, it taps other AMMs’ liquidity to be able to settle the traders’ orders. Gnosis Protocol v2 can be connected to any on-chain liquidity sources and can therefore enjoy the benefits of concentrating the fragmented liquidity across decentralized finance.

annamsgeorge commented:
> <img alt="Screenshot 2021-04-20 at 19 32 34" width="348" src="https://user-images.githubusercontent.com/33424780/115476123-7423a880-a241-11eb-9d62-273f1992108b.png">
> 
> I would add a bit more space before "Protocol" because it's a section header



Maybe before all section headers (General, Protocol, Trading) the space could be a tiny bit larger to differentiate it from sub-topics?

annamsgeorge commented:
<img width="650" alt="Screenshot 2021-04-20 at 19 53 39" src="https://user-images.githubusercontent.com/33424780/115477533-67ed1a80-a244-11eb-986e-44760134664e.png">


Here is another mistake. 

We don't subsidize all gas costs, only 90%.

It's also not clear to me whether it's referring to the protocol fee or the gas fee and which one is how much based on this. The last sentence claims it's about the gas fee (but we dont subsidize 100%) and its unclear how much the protocol fee is. I think this needs to be adjusted a bit, I will come back with a clearer sentence tomorrow. (when u read this today)

annamsgeorge commented:
<img width="727" alt="Screenshot 2021-04-20 at 19 58 22" src="https://user-images.githubusercontent.com/33424780/115477821-1002e380-a245-11eb-8755-1080bbaaa73e.png">


Can we please adjust the question to only say 

"How is CowSwap able to offer better prices?"

annamsgeorge commented:
I am sorry I noticed a few more things only now 
<img width="271" alt="Screenshot 2021-04-20 at 20 02 59" src="https://user-images.githubusercontent.com/33424780/115478111-b3ec8f00-a245-11eb-96a3-ac6803e351cb.png">

annamsgeorge commented:
> <img alt="Screenshot 2021-04-20 at 19 53 39" width="650" src="https://user-images.githubusercontent.com/33424780/115477533-67ed1a80-a244-11eb-986e-44760134664e.png">
> 
> Here is another mistake.
> 
> We don't subsidize all gas costs, only 90%.
> 
> It's also not clear to me whether it's referring to the protocol fee or the gas fee and which one is how much based on this. The last sentence claims it's about the gas fee (but we dont subsidize 100%) and its unclear how much the protocol fee is. I think this needs to be adjusted a bit, I will come back with a clearer sentence tomorrow. (when u read this today)

Update, here's the new text for this question:

The fee consists of the "base cost to execute the trade" and the "protocol fee" (although it is only exposed to the user as one fee). As a user, you are only signing a message to submit your trade and the underlying solver will end up submitting the transaction for you. Essentially you are paying this "base cost to execute the trade" aka "gas costs" with your sell token and the cost is already included in your price estimation. The protocol is currently subsidizing 90% of the gas cost, while the protocol fee is currently switched off.

biocom commented on src/custom/components/Page/index.tsx:
```diff
@@ -58,9 +59,31 @@ export const Content = styled.div`
     margin: 24px auto;
   }
 
+  #table-container {
```
Where is this used?

anxolin replied:
In the cookie policy, I moved it out of the page, since it's not general enough https://github.com/gnosis/gp-swap-ui/pull/470/commits/7312ee53860fa3acf9e956ad52d58ff3994111b5

anxolin commented:
I believe I addressed the issues

@annamsgeorge @biocom merging this so it's also available in the code freeze for testing

We will probably do small improvements now, but we will do it in a separate PR. So write if there's any not addressed issue or there's something else.
